K8s/soda-secret.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: soda-multi-cloud
+  name: soda-secret
```

Comment:
Suggestion: "soda-IAM-secret" may be more appropriate.

Reply:
no functional impact, skip